DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/ConfirmOverwriteSelfAssessment.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.SelfAssessments
+@using DigitalLearningSolutions.Web.Helpers
+@model ConfirmOverwrite
+@{
+    ViewData["Title"] = "Self Assessments - Confirm Overwrite";
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    var errorClass = Model.isChecked == false ? "nhsuk-input--error" : "";
+    var routeParamsForCancelLink = new Dictionary<string, string>();
+    routeParamsForCancelLink.Add("selfAssessmentId", Model.SelfAssessmentId.ToString());
+    routeParamsForCancelLink.Add("competencyNumber", Model.CompetencyNumber.ToString());
+}
+<div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+        @if (errorHasOccurred)
+        {
+            <vc:error-summary order-of-property-names="@(new[] { nameof(Model.isChecked) })" />
+        }
+        <h2 class="nhsuk-heading-xl">Overwrite confirmed self assessment result?</h2>
+        <p class="nhsuk-body-m word-break">
+            <span class="nhsuk-u-font-weight-bold">Competency: </span> @Model.CompetencyName
+        </p>
+        <form asp-action="ConfirmOverwriteSelfAssessment"
+              asp-route-competencyNumber="@Model.CompetencyNumber"
+              asp-route-competencyId="@Model.CompetencyId"
+              asp-route-competencyGroupId="@Model.CompetencyGroupId">
+
+            <vc:checkboxes label="Please check the checkbox to continue"
```

Comment:
We should use the vc:single-checkbox here. e.g.:

` <vc:single-checkbox asp-for="@nameof(Model.Confirm)"
                          label="I am sure that I wish to overwrite my confirmed self assessment result"
                          hint-text="this action will result in your new response being stored in place of the previous one and the confirmation status being reset." />`

Reply:
fixed